cms/src/common/business-form/business-form.jsx

```diff
@@ -0,0 +1,181 @@
+import style from "./business-form.module.scss";
+import React from "react";
+import PropTypes from "prop-types";
+import {Business} from "../../models/business";
+
+const RenderField = ({field, readOnly, value, onChangeFn, required = false}) => (
+  <>
+    {readOnly ?
+      <p className={style.readValue}>{value}</p>
+      :
+      <input id={`business-${field}`} onChange={onChangeFn} required={required}
+             value={value} name={field} className={style.input}/>
+    }
```

Comment:
Don't want you to change this, but inputs have a `readOnly` prop that makes them read only. Maybe you wanted it not to be an input, which is fine :+1:

Reply:
Yes, the idea behind of it being a <p> is that it's more flexible if we want to add a better style to it.